static/images/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
A manifest.json, great idea! Didn't think of that!
Does the name key need renaming though? Not sure exactly how they work.